README.md

```diff
@@ -4,7 +4,7 @@
 
 <h1><b>BookingBites App</b></h1>
  
-  <img src="reservations.png" alt="logo" width="400"  height="auto">
+  <img src="bookingbites.png" alt="logo" width="400"  height="auto">
```

Comment:
- [ ] OPTIONAL:  I suggest you add some tests in order to ensure your app's integrity. 👍

Reply:
Hello @Meltrust.  Thank you very much for the review, but I think we have a misunderstanding here...we sent to review the **splash-page branch** and we weren´t clear about it in the PR, our mistake, sorry for that.  So, I am pretty sure you reviewed the **dev branch**, that´s the reason you couldn´t see the login form.  Kindly, if it's not too much to ask, could you please review the project in the **splash-page branch** to check for any additional issues that require fixing?

Reply:
Thanks for your inquiry, I'm trying to communicate on Slack. 👍

Reply:
@JuanDBta According to your inquiry, the original review was completely redone and revised from scratch based on the `splash-page` branch.

Thanks for giving us the correct branch heads-up. 👍

Reply:
Thank you @Meltrust, great review!! We are on it now...sorry for the missing communication on Slack, it was midnight in my local time...have a nice week!